python/ray/_raylet.pyx

```diff
@@ -1421,6 +1427,40 @@ cdef class CoreWorker:
         logger.warning("Local object store memory usage:\n{}\n".format(
             message.decode("utf-8")))
 
+    cdef python_scheduling_strategy_to_c(
+            self, python_scheduling_strategy,
+            CSchedulingStrategy *c_scheduling_strategy):
+        cdef:
+            CPlacementGroupSchedulingStrategy \
+                *c_placement_group_scheduling_strategy
+        assert python_scheduling_strategy is not None
+        if python_scheduling_strategy == "DEFAULT" or \
+                isinstance(python_scheduling_strategy,
+                           DefaultSchedulingStrategy):
+            c_scheduling_strategy[0].mutable_default_scheduling_strategy()
+        elif isinstance(python_scheduling_strategy,
+                        PlacementGroupSchedulingStrategy):
+            c_placement_group_scheduling_strategy = \
+                c_scheduling_strategy[0] \
+                .mutable_placement_group_scheduling_strategy()
+            c_placement_group_scheduling_strategy[0].set_placement_group_id(
+                python_scheduling_strategy
+                .placement_group.id.binary())
+            c_placement_group_scheduling_strategy[0] \
+                .set_placement_group_bundle_index(
+                    python_scheduling_strategy.placement_group_bundle_index)
+            c_placement_group_scheduling_strategy[0]\
+                .set_placement_group_capture_child_tasks(
+                    python_scheduling_strategy
+                    .placement_group_capture_child_tasks)
+        else:
+            raise ValueError(
+                f"Invalid scheduling_strategy value "
+                f"{python_scheduling_strategy}. "
+                f"Valid values are [\"DEFAULT\""
+                f" | DefaultSchedulingStrategy"
+                f" | PlacementGroupSchedulingStrategy]")
+
```

Comment:
Minor, but we can do `RAY_CHECK(c_scheduling_strategy != nullptr)`?

Reply:
Added the check to core_worker.cc so it benefits all languages.